packages/m2m-gateway/test/mockUtils.ts

```diff
@@ -47,6 +48,27 @@ export function getMockedApiDelegation({
   };
 }
 
+export function getMockedApiTenant(): WithMetadata<tenantApi.Tenant> {
+  return {
+    data: {
+      id: generateId(),
+      attributes: generateMock(z.array(tenantApi.TenantAttribute)),
+      externalId: {
+        origin: generateMock(z.string()),
+        value: generateMock(z.string()),
+      },
+      name: generateMock(z.string()),
+      createdAt: new Date().toISOString(),
+      kind: tenantApi.TenantKind.Values.GSP,
+      mails: [],
+      features: [],
```

Comment:
Let's generate some of these as well as done for attributes
```suggestion
      mails: generateMock(z.array(tenantApi.Mail)),
      features: generateMock(z.array(tenantApi.TenantFeature)),
```